apps/wallet/src/features/Explorer/Highlights/IndicatorDot.tsx

```diff
@@ -0,0 +1,80 @@
+import type { FC } from 'react';
+import type { ViewStyle } from 'react-native';
+import { StyleSheet, TouchableOpacity } from 'react-native';
+import type { SharedValue } from 'react-native-reanimated';
+import Animated, {
+	interpolate,
+	useAnimatedStyle,
+	withTiming,
+} from 'react-native-reanimated';
+
+const AnimatedHoverable = Animated.createAnimatedComponent(TouchableOpacity);
+
+type IndicatorDotStyleProps = ViewStyle;
+
+interface IndicatorDotProps {
+	index: number;
+	setActiveIndex: (index: number) => void;
```

Comment:
do not use `setState` name for a function props, use `onPress` or sth else instead